docs/dev/env_devscripts.md

```diff
@@ -32,7 +32,7 @@ Go to https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/, clic
 right, copy the login command, extract the token from the command and use it to set `CI_TOKEN` in `config_$USER.sh`.
 
 #### Get pull secret
-Save the secret obtained from [cloud.openshift.com](https://cloud.redhat.com/openshift/install/pull-secret) to `pull_secret.json`.
+Convert the secret obtained from [cloud.openshift.com](https://cloud.redhat.com/openshift/install/pull-secret), originally in text format, into JSON format and save it as `pull_secret,json`. 
```

Comment:
Nope apparently it's txt, if you use the other option clicking "copy" it will copy the data (txt) into clipboard, I too didn't give it much thought, created an empty pull_secret.json file and pasted clipboard data(txt). Only when things failed again and again I started to investigate/debug, figured out the missing conversion. Thats why I figured I should mention this issue.

Reply:
OK, thanks @tshefi.